helpers/validatelockcreation.js

```diff
@@ -0,0 +1,111 @@
+function validateCommonInputs(inputs, validationErrors) {
+  // rearrange in order of CreatedLockFields to make sure everything gets validated
+  if(inputs.Shared != false && inputs.Shared != true) {
+    validationErrors.push("Shared is not valid");
+  }	// enforced by schema?
+
+  // LockName not required by schema, and a TypeError occurs if length is used on undefined
+  // TODO: ??? Should lockName be required to not be null in db, models, schema, etc. ???
```

Comment:
OK.  We will still need the code change that follows this comment, since if the name is allowed to be null and the validation tries to check the length of the null name, an error will result.